truffle.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  networks: {
+    local: {
+      host: "localhost",
+      port: 8545,
+      network_id: "*",
+      websockets: true,
+    },
+  },
+  compilers: {
+    solc: {
+      version: "0.7.6", // Fetch exact version from solc-bin (default: truffle's version)
+    },
+  },
+}
```

Comment:
It feels wrong to have both hardhat and truffle in the same project. 😬 

What is the problem with the package structure? Can we instead implement an adapter that would support both structures in the code importing them?

cc @nkuba given that he has more experience with Hardhat deployments.

Reply:
I don't like having both hardhat and truffle as they require maintaining two different configurations and migrations scripts.

A while ago Hardhat was not producing migration artifacts containing deployment details, e.g. network, address, or transaction hash. That's why we had to create our [own tool for mainnet migrations](https://github.com/keep-network/keep-migrations). I'm not sure if anything has changed in that matter. If we don't find any plugin or hardhat's feature that supports what we need regarding produced artifacts I'm afraid we need to craft the solution on our own. The advantage of doing this is the possibility to transition from truffle to hardhat in all other modules.

Reply:
Maybe https://github.com/wighawag/hardhat-deploy is something we need?

Reply:
Yes, I've seen it and played with it. The hardhat returned artifact in this format:
```
{
 abi: [<abi here>]
 address: "<address here>"
}
```

I'll dig into it and we'll see if we can add more deployment details.

Reply:
> The advantage of doing this is the possibility to transition from truffle to hardhat in all other modules.

This would be so nice. Switching to all hardhat + ethers :yum: